relay-server/src/services/processor.rs

```diff
@@ -3073,6 +3056,117 @@ impl Service for EnvelopeProcessorService {
     }
 }
 
+#[cfg(feature = "processing")]
+fn enforce_quotas<'a, G: Group, P: GroupPayload<'a, G>>(
```

Comment:
Temporarily duplicated, to make it work for both old and new processing mechanism.